tests/unit/cluster/cluster-reliable-meet.tcl

```diff
@@ -0,0 +1,79 @@
+# make sure the test infra won't use SELECT
+set old_singledb $::singledb
+set ::singledb 1
+
+tags {tls:skip external:skip cluster} {
+
+set base_conf [list cluster-enabled yes]
+start_multiple_servers 2 [list overrides $base_conf] {
+
+test "Cluster nodes are reachable" {
+    for {set id 0} {$id < [llength $::servers]} {incr id} {
+        # Every node should be reachable.
+        wait_for_condition 1000 50 {
+            ([catch {R $id ping} ping_reply] == 0) &&
+            ($ping_reply eq {PONG})
+        } else {
+            catch {R $id ping} err
+            fail "Node #$id keeps replying '$err' to PING."
+        }
+    }
+}
+
+test "Before slots allocation, all nodes report cluster failure" {
+    wait_for_cluster_state fail
+}
+
+set CLUSTER_PACKET_TYPE_MEET 2
+set CLUSTER_PACKET_TYPE_NONE -1
+
+set b 0
+set a 1
+
+test "Cluster nodes haven't met each other" {
+    assert {[llength [get_cluster_nodes $a]] == 1}
+    assert {[llength [get_cluster_nodes $b]] == 1}
+}
```

Comment:
```suggestion
test "Cluster nodes haven't met each other" {
    assert {[llength [get_cluster_nodes 1]] == 1}
    assert {[llength [get_cluster_nodes 0]] == 1}
}
```
If you much prefer a and b, I'm okay with, but it seems just as useful to just call them 0 and 1 to me.

Reply:
No strong preference. Will stick to 0/1.